tiledb/bioimg/converters/ome_tiff.py

```diff
@@ -139,6 +140,127 @@ def _original_metadata(self, key: str, default: Any = None) -> Any:
             pass
         return default
 
+    @property
+    def image_metadata(self) -> Dict[str, Any]:
+        metadata: Dict[str, Any] = {}
+        color_generator = iter_color(np.dtype(np.uint8))
+
+        if self._metadata and self._tiff.is_ome:
+            if isinstance(self._metadata["OME"]["Image"], list):
+                image = self._metadata["OME"]["Image"][0]["Pixels"]
+            else:
+                image = self._metadata["OME"]["Image"]["Pixels"]
+
+            channels = []
+
+            for idx, channel in enumerate(
+                image["Channel"]
+                if isinstance(image["Channel"], list)
+                else [image["Channel"]]
+            ):
+                if "SamplesPerPixel" in channel and channel["SamplesPerPixel"] != 1:
+                    for i in range(channel["SamplesPerPixel"]):
+                        channel_metadata = {
+                            "ID": channel["ID"] if "ID" in channel else f"{idx}-{i}",
+                            "Name": channel["Name"]
+                            if "Name" in channel
+                            else f"Channel {idx}-{i}",
+                            "Color": next(color_generator),
+                        }
+
+                        channels.append(channel_metadata)
+                else:
+                    channel_metadata = {
+                        "ID": channel["ID"] if "ID" in channel else f"{idx}",
+                        "Name": channel["Name"]
+                        if "Name" in channel
+                        else f"Channel {idx}",
+                        "Color": get_rgba(channel["Color"])
+                        if "Color" in channel
+                        else next(color_generator),
+                    }
+                    if "EmissionWavelength" in channel:
+                        channel["EmissionWavelength"] = length_converter(
+                            channel["EmissionWavelength"],
+                            channel["EmissionWavelengthUnit"]
+                            if "EmissionWavelengthUnit" in channel
+                            else "nm",
+                            "nm",
+                        )
```

Comment:
Try to take advantage of the `get` function for your dictionaries to avoid unnecessary `if-else` and make the code more readable.